src/components/AboutPage/AboutPage.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { CenterFocusStrong } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles({
+  aboutBackground: {
+    backgroundImage: 'linear-gradient(137deg, rgba(229,170,255,1) 0%, rgba(249,236,255,1) 100%)',
+    backgroundSize: '400% 400%',
+    animation: '$gradient 15s ease infinite',
+  },
+  '@keyframes gradient': {
+    '0%': {
+      backgroundPosition: '0% 50%',
+    },
+    '50%': {
+      backgroundPosition: '100% 50%',
+    },
+    '100%': {
+      backgroundPosition: '0% 50%',
+    },
+  },
+  title: {
+    fontSize: '44px',
+    lineHeight: '56px',
+    fontWeight: 700,
+  },
+  subtitle: {
+    fontWeight: 700,
+    fontSize: '23px',
+  },
+  cardstyle: {
+    width: '40%',
+    height: 400,
+    marginLeft: '30%',
+    marginRight: '30%',
```

Comment:
These values add weird spacing on mobule view ( I think its these values but it could be other) : 
![image](https://user-images.githubusercontent.com/14894196/104180387-b99c3280-5415-11eb-846a-8933362ae582.png)

Better just put the content into a <Container maxWidth="lg"> element to take care of the margins on Desktop view